README.md

```diff
@@ -1,70 +1,178 @@
-You may also see any lint errors in the console.
+> Soccer Hub is a football website that allows you to watch recent official highlights of the Premier League, Champions League, La Liga, Serie A, Europa League and many more matches of your favorite football leagues. It is built using React & Redux and uses ScoreBat recent feed free video API
 
-### `npm test`
+## 🛠 Built With <a name="built-with"></a>
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+### Tech Stack <a name="tech-stack"></a>
 
-### `npm run build`
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li>ReactJS</li>
+    <li>Redux</li>
+    <li>Javascript</li>
+    <li>Bootstrap</li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Github</li>
+  </ul>
+</details>
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+<!-- Features -->
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+### Key Features <a name="key-features"></a>
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+- **Filter recent videos highlights by competition name**
+- **Display all recent highlights in the selected competition**
+- **Click on a highlight of choice to watch the official match highlight**
 
-### `npm run eject`
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+#### 🔭 Future Features <a name="future-features"></a>
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+- **Add animations and transitions**
+- **Implement additional pages**
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+<!-- LIVE DEMO -->
 
-## Learn More
+## 🚀 Live Demo <a name="live-demo"></a>
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+- <a href="https://soccer-hub.onrender.com" target="_blank">Live Demo Link</a>
+  
+## 🚀 Project Presentation Video <a name="live-demo"></a>
```

Comment:
- [ ] Please make sure to add a valid link to your presentation video. 🙏

Reply:
@MahmoudBakr23 Thank you for the feedback.